### ISSUE TITLE
The advice from vuejs-accessibility/accessible-emoji appears to lead to a degraded experience with assistive technologies

### ISSUE THREAD

**Issue by slaweet** [vue-a11y/eslint-plugin-vuejs-accessibility#300]
The problem is analogical to https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/issues/627, which contains all the details.

TL;DR vuejs-accessibility/accessible-emoji rule should be deprecated and removed from recommended rule lists

**Comment by kddnewton** [comment]
Nice. I'll deprecate it as well.

**Comment by slaweet** [comment]
Cool. Thank you for such a quick fixing.

Though I think it should also be removed from https://github.com/vue-a11y/eslint-plugin-vuejs-accessibility/blob/fde0bbc95455a3da400d546dd0ab82bcc4ecc5f0/src/configs/recommended.ts#L15
